migrations/Version202201010708595959_generis.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\generis\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\reporting\Report;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202201010708595959_generis extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Synchronize models to add new "Is Statistical" class property';
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
```

Comment:
This is used to add new rows to the statement table ( with subject `http://www.tao.lu/Ontologies/generis.rdf#is_statistical`) for all existing property objects, right?

Reply:
It syncs all ontology files with out database and cache